compiler/Eta/Main/DriverMkDepend.hs

```diff
@@ -279,7 +279,7 @@ insertSuffixes
         :: FilePath     -- Original filename;   e.g. "foo.o"
         -> [String]     -- Suffix prefixes      e.g. ["x_", "y_"]
         -> [FilePath]   -- Zapped filenames     e.g. ["foo.x_o", "foo.y_o"]
-        -- Note that that the extra bit gets inserted *before* the old suffix
+        -- Note that the extra bit gets inserted *before* the old suffix
```

Comment:
This is the edge case. `that that` might be intended, but it didn't feel necessary, so I went with dropping one.